src/inputs/TagsEditor/TagsEditor.js

```diff
@@ -118,7 +119,7 @@ export const TagsEditor = (props) => {
           container
           sx={{ flexDirection: 'row', alignItems: 'center', justifyContent: 'flex-start', flexWrap: 'nowrap' }}
         >
-          <Typography variant="body1" className={classes.label} sx={{ pr: 1 }}>
+          <Typography className={classes.label} sx={{ ...headerStyle, pr: 1 }}>
```

Comment:
I'm not sure if this is correct, but it feels like the color is applied twice, by the `className` and within the `sx` prop, or am I missing something?
If it is applied twice, then maybe we could remove the `className` to use a single, more configurable, mechanism?